frontend/src/app/services/event-list.ts

```diff
@@ -192,17 +173,17 @@ export const eventTriggers: EventTrigger[] = [
         help: 'Opatření “uzavření škol” opět vyžaduje další náklady a snižuje společenskou stabilitu.',
       },
     ],
-    condition: (s: DayState) => seasonalDateDiff(s.date, '09-01') === 0,
-    timeout: 360, // Happens every year
+    condition: (s: DayState) => dateDiff(s.date, '2020-09-01') === 0,
+    timeout: 365, // Happens every year
   },
   {
     events: [
       {
         title: 'Virus se v teplém podnebí hůř šíří. Vědci předpokládají zpomalení pandemie.',
       },
     ],
-    condition: (s: DayState) => dateBetweenTrigger(s.date, '05-20', '06-14'),
-    timeout: 365, // Happens every year
+    condition: (s: DayState) => dateBetweenTrigger(s.date, '2020-05-20', '2020-06-14'),
+    timeout: 365 + _.random(-10, 10), // Happens every year (with some variation)
```

Comment:
Now it doesn't really happen every year. So remove the timeouts?

Reply:
:+1: removed